### DISCUSSION TITLE
Welcome to better_flutter Discussions!

### DISCUSSION THREAD

**Discussion by adoontheway** [adoontheway/better_flutter#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Uday-kiran9147** [comment]
While working on flutter applications everything works fine and in the middle of programming when I make changes, Sometimes My Emulator will not run the changes I have made, and then after It Completely stops Running flutter code, This makes me stop development. CAN ANYONE  SOLVE my problem?
